scripts/dropbox/migrate_to_external_accounts.py

```diff
@@ -0,0 +1,223 @@
+import logging
+import os
+import sys
+import shutil
+
+from nose.tools import *  # noqa
+from modularodm import Q
+
+from framework.mongo import database
+from framework.transactions.context import TokuTransaction
+
+from website import settings
+dropbox_views_path = os.path.join(
+    settings.BASE_PATH,
+    'addons',
+    'dropbox',
+    'views'
+)
+# Remove old dropbox views directory and .pyc files
+if os.path.isdir(dropbox_views_path):
```

Comment:
It's generally a good idea to avoid import side-effects. Maybe move this to the `migrate` function?